### PR TITLE
[full ci]Preparation for moving to snapshot based CI builds

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Need to switch to entirely environment based variables in order to eventually support snapshots in the CI

This does not turn on snapshots yet though, need more work to be able to handle local builds and to test the CI builds better

### CODE REVIEW

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -104,10 +97,13 @@ Get Docker Params
...
     ${rest}  ${ext-ip} =  Split String From Right  ${ext-ip}
     ${ext-ip} =  Strip String  ${ext-ip}
-    Set Suite Variable  ${ext-ip}  ${ext-ip}
+    Set Environment Variable  EXT-IP  ${ext-ip}
 
     ${rest}  ${vic-admin}=  Split String From Right  ${line}
-    Set Suite Variable  ${vic-admin}
+    Set Environment Variable  VIC-ADMIN  ${vic-admin}
+
+    Run Keyword If  ${port} == 2376  Set Environment Variable  VCH-PARAMS  -H ${dockerHost} --tls
```
Comment (anchal-agrawal) [inline tests/resources/Util.robot]: Should we use `Set Environment Variable` in both cases here?

File: tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot [hunk 2]
```diff
@@ -42,70 +42,70 @@ Created Network And Images Persists As Well As Containers Are Discovered With Co
...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start ${bridge-exited}
     Should Be Equal As Integers  ${rc}  0
 
-    Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  ${vch-ip}  10000
-    Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  ${vch-ip}  10001
+    Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  %{VCH-IP}  10000
+    Wait Until Keyword Succeeds  20x  5 seconds  Hit Nginx Endpoint  %{VCH-IP}  10001
 
-    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} create -it -p 10000:80 -p 10001:80 --name webserver1 nginx
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -it -p 10000:80 -p 10001:80 --name webserver1 nginx
```
Comment (emlin) [inline tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot]: Are we changing all ${params} to %{VCH-PARAMS}

Comment (mhagen-vmware) [inline tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot]: yep

**Comment by jzt** [comment]
LGTM

**Comment by anchal-agrawal** [comment]
LGTM

**Comment by mhagen-vmware** [comment]
.

**Comment by rajanashok** [comment]
LGTM

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**